src/main/java/edu/greenblitz/robotName/commands/prototypes/runMotors.java

```diff
@@ -0,0 +1,56 @@
+package edu.greenblitz.robotName.commands.prototypes;
+
+import com.ctre.phoenix.motorcontrol.TalonSRXControlMode;
+import edu.greenblitz.robotName.subsystems.Dashboard;
+import edu.greenblitz.robotName.subsystems.prototypes.Prototypes;
+import edu.greenblitz.robotName.utils.GBCommand;
+import edu.greenblitz.robotName.utils.tuneableNumber.TunableNumberManager;
+
+import static edu.greenblitz.robotName.subsystems.Dashboard.*;
+import static edu.greenblitz.robotName.subsystems.Dashboard.motorType.SPARK_MAX;
+import static edu.greenblitz.robotName.subsystems.Dashboard.motorType.TALON_SRX;
+
+public class runMotors extends GBCommand {
```

Comment:
for classes in java we use PascalCase
![pascalCase](https://github.com/GreenBlitz/CRRRRESCENDO/assets/95913534/3b7bf7a7-94cb-4d79-9f96-bb82fe00261e)